### DISCUSSION TITLE
Rename `TheCleanCoder` folder to `CleanCoder`

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#33] [General]
@Joseph-Cha 
동훈님만 유독 TheCleanCoder라는 폴더에 혼자 넣으셨더라고요.

CleanCoder/donghun 으로 넣어 주셔야 합니다.

다시 pull request를 주셔서 folder 이름 변경을 해 주세요.

**Comment by Joseph-Cha** [comment]
아 그러네요;;; 
책 제목이 영어로 The Clean Coder라 적혀있어서 책 제목이 저건 줄 알았습니다 
PR 날릴 때 수정해서 날릴게요

**Reply by jongfeel** [reply]
알겠습니다.
다시 보니 TheCleanCoder가 맞긴 한데, 다들 CleanCoder로 해서요. 동훈님이 맞춰주면 좋을 거 같아요.

**Reply by Joseph-Cha** [reply]
네넹~
